homeassistant/components/swiss_public_transport/strings.json

```diff
@@ -0,0 +1,27 @@
+{
+  "config": {
+    "error": {
+      "client": "Invalid request: Check at [stationboard](http://transport.opendata.ch/examples/stationboard.html) if your station names are valid",
+      "unknown": "Could not parse request response: Check at [stationboard](http://transport.opendata.ch/examples/stationboard.html) if your station names are valid"
```

Comment:
Both are kinda saying the same thing. I can imagine when you fill in non existent data it will only use of the two?